app/main.py

```diff
@@ -1,8 +1,59 @@
+from typing import Union
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int, clean_mark: int, brand: str) -> None:
```

Comment:
When you’re using a hanging indent, there must not be any arguments on the first line. The following example is not PEP 8 compliant

Reply:
It is PEP 8 compilant, cause self is just link on object, while others are parameters. Furthermore, here is just example from pep:
![image](https://user-images.githubusercontent.com/107141441/216962843-0a898498-2208-448e-8707-cc5b948ac963.png)
But it's just simple case from functions, while there are much more complicated cases, like in python modules:
![image](https://user-images.githubusercontent.com/107141441/216963131-ba94ca6f-5772-4d0a-b8cf-dbb6bdf17545.png)
The only thing I'm not totally sure is - either I need or not have all other parametres in hanging way, but black just reformated file this way, so all questions to https://pypi.org/project/black/